route/driver.js

```diff
@@ -1,38 +1,38 @@
 import express from 'express';
 import fs from 'fs';
+import path from 'path';
 
 const driver = express.Router();
-//get the list of driv 
-driver.get('/drivers', (req,res) => {
-	res.send({type: 'GET'});
+// get the list of driver
+driver.get('/drivers', (req, res) => {
+  res.send({ type: 'GET' });
 });
-//add a driver
-driver.post('/drivers', (req,res) => {
-	fs.readFile(__dirname+'/../Database/index.json','utf8', (err, data) => {
-		if (err){
-        	console.log(err);
-    	} else {
-    		let obj = JSON.parse(data);
-    		console.log("drivers = "+ req.body.id);
-    		//let id = "user" + (Object.keys(obj.users).length + 1);
-			obj.drivers[req.body.id] = req.body;
-			console.log("driver id = "+obj.drivers[req.body.id]);
-			let json = JSON.stringify(obj);
-			fs.writeFile(__dirname+'/../Database/index.json', json, 'utf8', err => {
-				if(err) throw err;
-				console.log("file has been saved");
-			});
-			res.send(req.body.id);
-    	}
-	});
+// add a driver
+driver.post('/drivers', (req, res) => {
+  fs.readFile(path.json(__dirname, '/../Database/index.json'), 'utf8', (err, data) => {
+    if (err) {
+      console.log(err);
+    } else {
+      const obj = JSON.parse(data);
+      // let id = "user" + (Object.keys(obj.users).length + 1);
+      obj.drivers[req.body.id] = req.body;
+      console.log(obj.drivers[req.body.id]);
+      const json = JSON.stringify(obj);
+      fs.writeFile(path.join(__dirname, '/../Database/index.json'), json, 'utf8', (er) => {
+        if (er) throw er;
+        console.log('file has been saved');
+      });
+      res.json(req.body.id);
+    }
+  });
```

Comment:
separate your routes from your methods, have a routes file, and a controllers folder contains the methods for your routes